pilot/pkg/proxy/envoy/v2/debug.go

```diff
@@ -121,21 +119,21 @@ func Syncz(w http.ResponseWriter, req *http.Request) {
 // registryz providees debug support for registry - adding and listing model items.
 // Can be combined with the push debug interface to reproduce changes.
 func (s *DiscoveryServer) registryz(w http.ResponseWriter, req *http.Request) {
-	_ = req.ParseForm()
-	w.Header().Add("Content-Type", "application/json")
-	svcName := req.Form.Get("svc")
-	if svcName != "" {
-		data, err := ioutil.ReadAll(req.Body)
-		if err != nil {
-			return
-		}
-		svc := &model.Service{}
-		err = json.Unmarshal(data, svc)
-		if err != nil {
-			return
-		}
-		s.MemRegistry.AddService(model.Hostname(svcName), svc)
-	}
+	//_ = req.ParseForm()
```

Comment:
This is useful for tests (including perf/bench) - should probably do a 'if s.MemRegistry != nil'

Reply:
But mem registry is always enabled in the InitDebug .. even in prod.